olive/passes/onnx/common.py

```diff
@@ -183,3 +185,29 @@ def model_proto_to_olive_model(
         onnx.checker.check_model(olive_model.model_path)
 
     return olive_model
+
+
+def get_lora_name_patterns() -> List[str]:
+    """Get the node name patterns for lora modules."""
+    return [
+        f".*[./]{name}[./]{matmul}$"
+        for name in ["default", "default_1", "lora_A", "lora_B"]
+        for matmul in ["MatMul", "MatMul_Q4"]
+    ]
```

Comment:
Aren't the nodes usually numbered i.e. <prefix>_0, <prefix_1, and so on. With the '$' at the end it might not match the numbered nodes.

Reply:
I haven't found any those are numbered. this are the existing patterns we know of and it works (this method was just copied over from a different file we have been using actively.